docs/runbooks/OutdatedVirtualMachineInstanceWorkloads.md

```diff
@@ -68,9 +68,9 @@ If a VMI is not live-migratable and if `runStrategy: always` is set in the
 corresponding `VirtualMachine` object, you can update the VMI by manually
 stopping the virtual machine (VM):
 
-```bash
-$ virctl stop --namespace <namespace> <vm>
-```
+  ```bash
+  $ virctl stop --namespace <namespace> <vm>
```

Comment:
Not related to this PR, but there is a typo in `virtctl` (spelled `virctl` currently)

Reply:
updated